{"name": "Проверка логина несуществующим курьером", "status": "passed", "description": "Пытаемся залогиниться несуществующим аккаунтом курьера, получаем ошибку 404 и соотвествующее сообщение", "steps": [{"name": "Попытка залогиниться в систему с несуществующими логином и паролем", "status": "passed", "start": 1714996472968, "stop": 1714996473198}], "start": 1714996472968, "stop": 1714996473199, "uuid": "417d135c-0522-466d-b4a5-6b451821a83a", "historyId": "b5a007415a4fe82fdd455da5981fbf07", "testCaseId": "b5a007415a4fe82fdd455da5981fbf07", "fullName": "tests.test_login_courier.TestLoginCourier#test_no_such_username_and_password", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login_courier"}, {"name": "subSuite", "value": "TestLoginCourier"}, {"name": "host", "value": "DESKTOP-M527MP1"}, {"name": "thread", "value": "36020-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login_courier"}]}